settings.yml

```diff
@@ -1,7 +1,8 @@
-folderPath: /opt/takipi-storage/storage
-maxUsedStoragePercentage: 0.95
+folderPath: /home/david/temp/temp/1523379396
```

Comment:
Don't change this for the prod release version :)

Reply:
wow thats bad, thanks for seeing it..